TypeCobol/Compiler/Diagnostics/Resources/DiagnosticMessages.csv

```diff
@@ -19,7 +19,7 @@ Category;Code;Severity;MessageTemplate;Document;PageNumber;ReferenceText
 2;18;1;Invalid floating point numeric literal format : the exponent is represented by an E followed by an optional sign and one or two digits (E+99);1;38;"Format for floating-point literals : ('+' | '-')? mantissa 'E' ('+' | '-')? exponent. The sign is optional before the mantissa and the exponent; if you omit the sign, the compiler assumes a positive number. The mantissa can contain between one and 16 digits. A decimal point must be included in the mantissa. The exponent is represented by an E followed by an optional sign and one or two digits."
 2;19;2;The number of characters in an hexadecimal alphanumeric literal should be a multiple of 2;1;36;An even number of hexadecimal digits must be specified. Two hexadecimal digits represent one character in a single-byte character set (EBCDIC or ASCII). 
 2;20;2;The number of characters in an hexadecimal national literal should be a multiple of 4;1;41;The number of hexadecimal digits must be a multiple of four. Each group of four hexadecimal digits represents a single national character and must represent a valid code point in UTF-16. 
-2;21;1;The first non blank character of this continuation line must be {0} (continuation of the alphanumeric literal on the previous line);1;55;If the continued line contains an alphanumeric or national literal, the continuation line must contain a hyphen in the indicator area, and the first nonblank character must be a quotation mark
+2;21;1;The delimiter character of this continuation line is different of the initial delimiter of the alphanumeric literal;1;55;If the continued line contains an alphanumeric or national literal, the continuation line must contain the same quotation marks
```

Comment:
English : "is different from"

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1604/commits/5de0690f865a31192983ee6452e96a892455d316